ios/AppDelegate.m

```diff
@@ -21,6 +21,10 @@
 @implementation AppDelegate
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
```

Comment:
Crashlytics was set up in #11 , so changes here can be removed

Reply:
It didn't look like Crashlytics was set up for iOS in this repo because the iOS apps in the Pathcheck-GAEN-dev project in Firebase are not enabled for Crashlytics.  Now that I look at  #11 I see I was wrong. I went down this path to get a GoogleService-info.plist to copy because the error in the CI test indicated something wrong with that.  If the file is being copied in the fetch-ha script, I can't run that script because I don't have permission.  I will back out the changes and see if the problem can be solved another way.